interfaces/prompting/requestprompts/requestprompts.go

```diff
@@ -530,7 +467,6 @@ func (pdb *PromptDB) Close() error {
 
 	// Clear all outstanding prompts
 	pdb.perUser = nil
-	unix.Munmap(pdb.maxIDMmap)
-	pdb.maxIDMmap = nil // reset maxIDMmap to nil so we know pdb has been closed
+	pdb.maxIDMmap.Munmap()
```

Comment:
Yes, also I moved it above the notices so if an error occurs, nothing happens, and the caller can potentially retry.

Reply:
Though perhaps this is a bad idea, if the kernel fails to munmap, it probably means the mmap has already been closed, since we shouldn't be messing with the address internally.